server/konnectors/linkedin.coffee

```diff
@@ -305,7 +327,7 @@ saveContact  = (newContact, entries, callback) ->
 
     # Case where the contact already exists and where it was imported from
     # Linkedin.
-    if entries.cozyContactsByAccountIds[newContact.id]?
+    if entries.cozyContactsByAccountIds[linkAccount.id]?
```

Comment:
The array cozyContactsByAccountIds references each contact by its account id and not the contact id. The last commit (newContact.id) leades to re-update all contacts each time.